### ISSUE TITLE
train_df, _ 

### ISSUE THREAD

**Issue by elyselam** [diogojapinto/ctr-prediction#3]
Hello,
Thanks for sharing the notebook :) Can you please clarify what this snippet of code does? 
Does the underscore in 
train_df, _ = derive_time_features(train_df)
tells the return function to ignore start_hour?

<img width="805" alt="screen shot 2018-07-21 at 8 30 18 pm" src="https://user-images.githubusercontent.com/30991021/43041984-df87684e-8d24-11e8-81a8-9c15c08ecd95.png">

**Comment by diogojapinto** [comment]
That is correct, I don't care about the second return value. You can check a detailed explanation for example here: https://hackernoon.com/understanding-the-underscore-of-python-309d1a029edc
